lib/internal/process.js

```diff
@@ -65,6 +65,13 @@ function setupConfig(_source) {
     if (value === 'false') return false;
     return value;
   });
+  if ( process.config.variables.v8_enable_i18n_support && global.hasOwnProperty('Intl') 
```

Comment:
Thanks. I'll drop the check for `process.config.variables.v8_enable_i18n_support` also. For this code's purpose, checking `Intl` is enough. One fewer dependency on an internal v8 variable is better.